IPython/lib/latextools.py

```diff
@@ -161,9 +161,26 @@ def latex_to_png_dvipng(s, wrap, color='Black', scale=1.0):
 
             resolution = round(150*scale)
             subprocess.check_call(
-                ["dvipng", "-T", "tight", "-D", str(resolution), "-z", "9",
-                 "-bg", "transparent", "-o", outfile, dvifile, "-fg", color],
-                 cwd=workdir, stdout=devnull, stderr=devnull)
+                [
+                    "dvipng",
+                    "-T",
+                    "tight",
+                    "-D",
+                    str(resolution),
+                    "-z",
+                    "9",
+                    "-bg",
+                    "Transparent",
+                    "-o",
+                    outfile,
+                    dvifile,
+                    "-fg",
+                    color,
```

Comment:
@Carreau: If you want to reformat, I would do it like this:

```suggestion
                    "dvipng",
                    "-T", "tight",
                    "-D", str(resolution),
                    "-z",  "9",
                    "-fg", color,
                    "-bg", "Transparent",
                    "-o", outfile,
                    dvifile,
```

Reply:
The reformatting is automatic with darker (which is basically black, but only on changed lines). 
So even if that may looks better, darker would reformat anyway.

Reply:
Ok. Then we have to live with it.